OWNERS

```diff
@@ -1,15 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
 approvers:
-- jsafrane
-- lpabon
-- msau42
-- saad-ali
-- sbezverk
-- vladimirvivien
+- kubernetes-csi-approvers
+
 reviewers:
+- kubernetes-csi-reviewers
+
+emeritus_reviewers:
```

Comment:
Missing the following from emeritus_reviewers and emeritus_approvers list:
lpabon
sbezverk
vladimirvivien